.github/workflows/build_and_generate_artifact.yml

```diff
@@ -0,0 +1,20 @@
+name: Generate default_config.zip
+
+on:
+ push:
+    branches:
+      - master
+      - BAH-*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Generate Artifact
+      run : ./scripts/package.sh
+    - name: Upload Artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: default_config
+        path: target/default_config.zip
```

Comment:
which GHA is using this artifact?

Reply:
Its been used by openmrs-module-bahmniapps, openmrs-distro-bahmni to build docker images